### DISCUSSION TITLE
新版？

### DISCUSSION THREAD

**Discussion by bigbug-dev** [the1812/Bilibili-Evolved#2336] [综合讨论 / General]
![image](https://user-images.githubusercontent.com/62877853/135132606-83982a94-77d3-4532-83f5-024d38f2bbb4.png)
第一次能用上整个屏幕，还挺舒服的😆

**Comment by bigbug-dev** [comment]
21:9的显示器，之前都是在两边有较大空余部分的

**Comment by the1812** [comment]
忘限制宽度了..
